### DISCUSSION TITLE
Avatto zigbee dimmer - in supported device list, paired, but no endpoints

### DISCUSSION THREAD

**Discussion by mogim0gi** [Koenkk/zigbee2mqtt#24660] [Devices]
Hei,

Just got a couple of these :
https://www.zigbee2mqtt.io/devices/ZDMS16-1.html

Connected, paired, however zigbee2nqtt doesnt give me desired result of controlling it:

![image](https://github.com/user-attachments/assets/9542ddfd-bcd8-471e-9467-8b34b3898e36)

What am I doing wrong ?
Zigbee2MQTT version - [1.41.0]

**Comment by munkdar** [comment]
I've got the same problem

**Comment by chris-1243** [comment]
> What am I doing wrong ?
Zigbee2MQTT version - [1.41.0]

The list of supported devices is based on the lastest stable version which is 2.2.1 nowadays.

I would update and take care of the breaking changes from 1.x to 2.x
